limit.go

```diff
@@ -48,25 +38,41 @@ type Limiter interface {
 	GetConnLimits() Limit
 }
 
-// BasicLimiter is a limiter with fixed limits.
-type BasicLimiter struct {
```

Comment:
removing this and the json config just flat out breaks lotus and ipfs... We should strive to avoid unnecessary breakage, and they are not incompatible with auto scaling -- they let the user explicitly specify limits.

Reply:
The JSON config is not getting removed. It is still possible to load static limits via JSON.